com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -41,20 +41,29 @@ public interface IUIWidgetsWindow {
         UIWidgetsWindowType getWindowType();
     }
 
+    public class ConfigurationSettings {
+        public static Dictionary<IntPtr, bool> _internalShowDebugLog = new Dictionary<IntPtr, bool>();
+    }
+
+    public class Configurations {
```

Comment:
rename:
_internalTextFonts => _textFonts
_showDebugLog => _debugMode

make the two variable private;

create new APIs to change the values of these configurations
create public getters for these configurations: (e.g., public bool enableDebug => _enableDebug;

note, it seems like we don't actually need a getter for _textFonts, instead we only need a getter like:
public object font => fontsToObject(_textFonts);